barcodes_generator_product/readme/DESCRIPTION.rst

```diff
@@ -1,12 +1,10 @@
 This module expands Odoo functionality, allowing user to generate barcode
 depending on a given barcode rule for Products.
 
-For example, a typical pattern for products is  "20.....{NNNDD}" that means
-that:
-* the EAN13 code will begin by '20'
+For example, a typical pattern for products is "8012345....." that means that:
```

Comment:
Dont remove variable pattern that is valid setting. But ypu could add your exemple. 

Thanks !

Reply:
Thanks for your feedback, I left the previous example and also added the new one.